LiteNetLib/NetSocket.cs

```diff
@@ -38,8 +38,11 @@ public short Ttl
 
         static NetSocket()
         {
-#if DISABLE_IPV6
+#if DISABLE_IPV6 || (ENABLE_IL2CPP && !UNITY_2018_3_OR_NEWER)
             IPv6Support = false;
+#elif ENABLE_IL2CPP && UNITY_2018_3_OR_NEWER
+            string version = Application.unityVersion;
+            IPv6Support = Socket.SupportsIPv6 && int.Parse(version.Remove(version.IndexOf('f')).Split('.')[2]) >= 6;;
```

Comment:
Crappy code, but needed, since IPv6 works with IL2CPP only with Unity 2018.3.6f1

Reply:
What about Unity 2019 and newer? I think this check would not pass.